src/components/button-toggle-group/button-toggle-group.test.js

```diff
@@ -348,4 +348,26 @@ context("Testing Button-Toggle-Group component", () => {
         });
     });
   });
+
+  describe("should make css changes when fullWidth prop is passed", () => {
+    it("container div should auto flex", () => {
+      CypressMountWithProviders(<ButtonToggleGroupComponent fullWidth />);
+
+      getDataElementByValue("button-toggle").should(
+        "have.css",
+        "flex",
+        "1 1 auto"
+      );
+    });
+
+    it("width of label should be 100% / 444px", () => {
+      CypressMountWithProviders(<ButtonToggleGroupComponent fullWidth />);
+
+      getDataElementByValue("button-toggle-label").should(
+        "have.css",
+        "width",
```

Comment:
this will fail locally, can you use like this:


```suggestion
buttonToggleLabelPreview().then(($el) => {
    useJQueryCssValueAndAssert($el, "width", 444);
});
```